subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperLoggingIntegrationTest.groovy

```diff
@@ -31,4 +34,39 @@ task emptyTask
         then:
         result.output.empty
     }
+
+    def "wrapper logs when there is a problem setting permissions"() {
+        given: "malformed distribution"
+        // Repackage distribution with bin/gradle removed so permissions cannot be set
+        def tempUnzipDir = temporaryFolder.createDir("temp-unzip")
+        distribution.binDistribution.unzipTo(tempUnzipDir)
+        distribution.binDistribution.delete()
+        assert tempUnzipDir.file("gradle-${distribution.version.version}", "bin", "gradle").delete()
+        tempUnzipDir.zipTo(distribution.binDistribution)
+        prepareWrapper(distribution.binDistribution.toURI())
+
+        when:
+        def failure = wrapperExecuter
+            .withTasks("emptyTask")
+            .withStackTraceChecksDisabled()
```

Comment:
:o: We don't strictly need this method call for the test case. May be something left from when you were testing it. Same comment below.

Reply:
Actually you're totally correct for this test, but not the 2nd one below, where the exception is rethrown and fails the build (and prints a stacktrace). This line is now removed